tools/generator/src/Generator/CreateFilesAndGroups.swift

```diff
@@ -214,6 +254,8 @@ extension Generator {
             // e.g. resources/App.strings
             let groupFilePath = localizedContainerFilePath.parent() + fileName
 
+            // TODO: Use `resolveFilePath`?
```

Comment:
leftover or a valid TODO?

Reply:
Valid. I couldn't find an easy way, fast, to do it because of the odd structure. Since it's a leaf, and not a source file, and most likely not at the root of a `new_local_repository`, I don't think it will matter to most people. Leaving the TODO to remind a future person about it though, in case they want to fix it.